README.md

```diff
@@ -1,2 +1,13 @@
 # Responsive-web-page
-Responsive design project
+
+The project is about applying responsive design techniques so that the website behaves differently depending on the size of the screen where the website is displayed. The website adapts to three views: Mobile, tablet, and large screen.
+
+https://user-images.githubusercontent.com/65791349/113003573-23a1b980-9173-11eb-8a31-7517f6a1b655.png
+
+# Built With
+
+HTML & CSS
+
+# Live Demo
+
+https://1v4n4.github.io/Responsive-web-page/
```

Comment:
## Improve the `README`
- [ ] We currently do need to have the name of the authors listed in the `README` and a screenshot to have a descriptive `README`. The format can also be improved further. Kindly use this [template](https://github.com/microverseinc/readme-template) as an inspiration.